python/pyarrow/tests/test_acero.py

```diff
@@ -115,6 +115,14 @@ def test_filter(table_source):
         FilterNodeOptions(None)
 
 
+def test_filter_all_rows():
```

Comment:
Can we add a check that the result is a `RecordBatch`? And maybe a test with the `Table` as an input for the empty filtering result so we are sure not to break anything there.

Reply:
Ah, I see there are existing tests in `test_compute.py`:
https://github.com/apache/arrow/blob/a5b2574208587464008127bbc85619562e6e85dd/python/pyarrow/tests/test_compute.py#L1399-L1435

Can we move it there?